dev/TabView/TestUI/TabViewPage.xaml.cs

```diff
@@ -44,7 +44,15 @@ public void IsCloseableCheckBox_CheckChanged(object sender, RoutedEventArgs e)
             }
         }
 
-        public void AddTabButton_Click(object sender, RoutedEventArgs e)
+        public void IsAddButtonVisibleCheckBox_CheckChanged(object sender, RoutedEventArgs e)
```

Comment:
>IsAddButtonVisibleCheckBox_CheckChanged [](start = 20, length = 39)

Why not just use x:Bind to bind these two together?  Or Binding with ElementName?

Reply:
I actually have no idea why I didn't think of that.